### PR TITLE
Cookie

### PR DESCRIPTION
**Pull request by jamesbhobbs**
**Overall change:** Move cookie tests, makes them reuseable and not tied to news

**Code changes:**
- makes them reuseable and not tied to news
- does little else :P 

NB: this doesnt make the cookie banner tests any better, but simply makes them not tied to 1 service.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by jamesbhobbs** [comment]
LGTM, cannot approve as I opened PR - NB these tests are risky to change, we should monitor CD carefully, likelihood of rollback is high.

File: cypress/integration/specialFeatures/cookieBanner.amp.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          });
+
+          it('should not show privacy & cookie banners once cookie banner declined, on reload', () => {
+            getPrivacyBanner(service).should('be.visible');
+            getCookieBanner(service).should('not.be.visible');
+
+            getPrivacyAccept(service).click();
+            getCookieReject(service).click();
+
+            cy.visit(`${config.news.pageTypes.articles}.amp`);
```
Comment (thekp) [inline cypress/integration/specialFeatures/cookieBanner.amp.js]: we still have `news` here ?

Comment (dr3) [reply cypress/integration/specialFeatures/cookieBanner.amp.js]: Ooh good spot

**Review by thekp — APPROVED** [review]

**Pull request merged**